src/Pos/Generator/Block/Mode.hs

```diff
@@ -167,6 +180,18 @@ mkBlockGenContext bgcParams@BlockGenParams{..} = do
         bgcTxpMem <- (,ignoreTxpMetrics) <$> mkTxpLocalData
         bgcDelegation <- mkDelegationVar @SscGodTossing
         return BlockGenContext {..}
+  where
+    -- Genesis utxo is needed only for boot era stakeholders
+    bgcGenesisUtxo =
```

Comment:
Wait-wait-wait, `GenesisUtxo` in `TestParams` is really genesis. But at this point we can have different Utxo in DB! That's why we need bootstrap stakeholders.